dist/src/main/config/gateway.yaml.template

```diff
@@ -47,6 +47,10 @@
       # This setting can also be overridden using the environment variable ZEEBE_GATEWAY_NETWORK_MINKEEPALIVEINTERVAL.
       # minKeepAliveInterval: 30s
 
+      # Sets the maximum size of the incoming and outgoing messages (i.e. commands and events).
+      # This setting can also be overridden using the environment variable ZEEBE_GATEWAY_NETWORK_MAXMESSAGESIZE.
+      # maxMessageSize: 4MB
```

Comment:
:wrench: May be refer to `ZEEBE_BROKER_NETWORK_MAXMESSAGESIZE` to indicate both configs must be changed to get the desired result.

Reply:
:+1: Good point